examples/twittermap/web/public/javascripts/map/pointmap.js

```diff
@@ -0,0 +1,347 @@
+angular.module('cloudberry.map')
+  .controller('pointMapCtrl', function($scope, $rootScope, $window, $http, $compile, cloudberry, leafletData, cloudberryConfig, Cache) {
+    // map type change handler
+    // initialize the map (styles, zoom/drag handler, etc) when switch to this map
+    // clear the map when switch to other map
+    $rootScope.$on('maptypeChange', function (event, data) {
+      if (cloudberry.parameters.maptype == 'pointmap') {
```

Comment:
Let's change it to pinmap. And I think these "countmap", "pinmap", and "heatmap" should be defined as a constant values like Java final string value. We are using these constants again and again.

Reply:
I am not sure about this. I thought pointmap shouldn't know the existence of other map type. Or maybe I misunderstand what you mean. We should talk about it.